### PR TITLE
fix: use wss protocol for production url

### PR DESCRIPTION
**Pull request by DikenMaharjan**

### CODE REVIEW

**Comment by DikenMaharjan** [comment]
This PR also contains some modification required for BouncyCastle to work in android.

File: src/main/kotlin/com/safehill/kclient/util/Utils.kt [hunk 1]
```diff
@@ -12,6 +12,7 @@ inline fun <T> runCatchingPreservingCancellationException(
     } catch (e: CancellationException) {
         throw e
     } catch (e: Exception) {
+        println("Error $e")
```
Comment (gennaro-safehill) [inline src/main/kotlin/com/safehill/kclient/util/Utils.kt:15]: What was the decision on logging in the framework?

**Review by gennaro-safehill — APPROVED** [review]
LGTM

File: src/main/kotlin/com/safehill/kclient/util/Utils.kt (hunk 1 above)
Comment (DikenMaharjan) [reply src/main/kotlin/com/safehill/kclient/util/Utils.kt:15]: I will have that implemented as well in the next PR.

**Pull request merged**